valerymo/integreatly-operator
# ServiceMonitors for Observability

# Issue link
https://issues.redhat.com/browse/MGDAPI-2680

# What
Makes all targets from AMO available in OO.

# Verification steps
* Checkout this PR
* Either wait until https://github.com/redhat-developer/observability-operator/pull/89 is merged or update the OO index to `quay.io/ckyrillo/observability-operator-index:v3.0.8`
  * https://github.com/valerymo/integreatly-operator/blob/oo-install/products/installation.yaml#L108
* Install RHOAM 
* Once the installation is complete:
  * Verify that the ServiceMonitors are created in the `redhat-rhoam-observability` namespace
  * Verify that the following metrics and probes are present on the Targets page in the Prometheus instance in the `redhat-rhoam-observability` namespace
   
![prometheus-targets](https://user-images.githubusercontent.com/80267718/136276175-53c0a586-dc3b-4db3-86f0-aa851b86180e.png)
